### DISCUSSION TITLE
Issue with windows 11 insider edition and your app

### DISCUSSION THREAD

**Discussion by minustee** [valinet/ExplorerPatcher#2704] [General]
So I am running an AMD ryzen x7 5800x. When I initially installed the explorerpatch. It bugged out and I had to reboot my PC and and try it again. I'm not sure what happened but I did this twice and it seemed to work. Then when I tried to run my AMD software (RyzenMaster) it said my CPU was incompatible... So I download AMD's driver software (AMDStoreMI) and try and install it. AMDstore launches and gives me an incompatible systems error... strange.. so I do a full windows 11 repair/reset. Reinstall AMD Ryzen and Store and it works.. Then I try and install explorerpatcher. Go to reboot my pc. PC now won't load into windows. Windows 11 Insider bluescreen/greenscreen of death states the boot drive is inaccessible. I try and run repair, repair diagnostics. Nothing works. Luckily I had just repaired windows 11 so I had a fresh recovery state to go back to. So here I am now. 
I like your app but now I have to see the pros and benefits of and if it even runs on my machine in normal windows 11 (not insider ed.)

OK forget this.. seems its an issue with AMDSTOREMI and not your app. I'll leave this up.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
